airgun/widgets.py

```diff
@@ -80,3 +82,35 @@ def search(self, value, result_locator=None):
         self.fill(value)
         self.search_button.click()
         return self.read(value, result_locator)
+
+
+class HorizontalNavigation(Widget):
+    """The Patternfly style horizontal top menu.
+
+    Use :py:meth:`select_item` to select the menu items. This takes IDs.
+    """
+    LEVEL_1 = '//ul[contains(@class, "navbar-menu")]/li/a[@id={}]'
+    LEVEL_2 = (
+        '//ul[contains(@class, "navbar-menu")]/li[./a[@id={}] and contains(@class, "open")]'
```

Comment:
We do follow pep8, in line lengths as well :) Please update PR with 79 chars length limit

Reply:
Because of locators, we bumped to 100, nobody has such small screens anymore, as you please :)

Reply:
My is small enough to fit exactly 2 tabs with 79 chars on the same screen ;)